Reflections/Week_15_Reflection.md

```diff
@@ -0,0 +1,34 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain) 
+  - I was able to create and add in a lot of details for the Unit Testing (shown in D5)
+  - Changed Cognative Walkthrough to vertical End-to-End testing. Did the outline and did one type of testing. (shown in D5)
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  - Finding a second device to record things since the app is on my phone. One of the professors gave me a camera and a tablet to allow me to do this now. The main difficulty now is finding time to do the rest of what is needed.
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  - film and do the rest of the End-to-End Testng (continuous, 2+ hours)
+    - Specific: I need to test the entire system with the goal of it to be focused on the turnability of the entire system.
+    - Measurable: i will be using a measuring tape and cones to help figure out how much it can turn. 
+    - Achievable: Sort of, the main conflict is finding time and the space. The space I have been using is the Berea park and it's parking lot. It isn't alwasy free to use.
+  - Test wheels with gravel. (30 minutes)
+    - Specific: I got to record the system trying to run over gravel. To see if it is possible, if so, how easily does it go over it. 
+    - Measurable: Yes, if it struggles at all then it is not an ideal surface for the system to run on. 
```

Comment:
How do you define struggling? Do you have a metric for what's struggling vs what's not for when you test your system on gravel?